### PR TITLE
Added 'FF2_SetClientDamage'

### PR DESCRIPTION
**Pull request by Nopied**
:D re-open

### CODE REVIEW

File: addons/sourcemod/scripting/freak_fortress_2.sp [hunk 1]
```diff
@@ -1023,6 +1023,7 @@ public APLRes:AskPluginLoad2(Handle:myself, bool:late, String:error[], err_max)
 	CreateNative("FF2_SetBossRageDamage", Native_SetBossRageDamage);
 	CreateNative("FF2_GetBossRageDistance", Native_GetBossRageDistance);
 	CreateNative("FF2_GetClientDamage", Native_GetClientDamage);
+	CreateNative("FF2_SetClientDamage", Native_SetDamage);
```
Comment (50Wliu) [inline addons/sourcemod/scripting/freak_fortress_2.sp]: Just one little thing: can you make this `Native_SetClientDamage` please?

**Comment by 50Wliu** [comment]
Ignore the failing Jenkins build.  I'm working on fixing that right now.

**Comment by 50Wliu** [comment]
--none--

**Comment by Nopied** [comment]
@50Wliu Yeah :D

**Comment by 50Wliu** [comment]
Thanks!

**Pull request merged**